Python/IP address info Finder/core/FileExporter.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# encoding: UTF-8
+
+"""
+    This file is part of IPGeoLocation tool.
+    Copyright (C) 2015-2016 @maldevel
+    https://github.com/maldevel/IPGeoLocation
+    
+    IPGeoLocation - Retrieve IP Geolocation information 
+    Powered by http://ip-api.com
+    
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
```

Comment:
This looks like you have entirely picked up the project from someone else. I would have surely appreciated if you took inspiration from others and tried out on you own but looks like you didnt even try